### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by LiuIhor**
It is my first project Tic-Tac-Toe

### CODE REVIEW

File: src/task1/Program.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+     * This method checks the player for a win
+     * */
+    public static boolean  checkWins(String[] symbols, Symbol symbol) {
+        boolean horizontal = (symbol.toString().equals(symbols[0]) && symbol.toString().equals(symbols[1]) && symbol.toString().equals(symbols[2]))
+                ||
+                (symbol.toString().equals(symbols[3]) && symbol.toString().equals(symbols[4]) && symbol.toString().equals(symbols[5]))
+                ||
+                (symbol.toString().equals(symbols[6]) && symbol.toString().equals(symbols[7]) && symbol.toString().equals(symbols[8]));
+
+        boolean vertical = (symbol.toString().equals(symbols[0]) && symbol.toString().equals(symbols[3]) && symbol.toString().equals(symbols[6]))
```
Comment (romaro488) [inline src/task1/Program.java]: Please move this logic to separate method

File: src/task1/README.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# SimpleTicTacToe
```
Comment (romaro488) [inline src/task1/README.txt]: Please move `Readme` to the project folder

File: src/task1/Program.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    }
+
+    /**
+     * This method counts game characters on the field
+     * */
+    private static int countSymbol(String[] symbols, Symbol symbol){
+        int count = 0;
+
+        for (String s : symbols) {
+            if (s.equals(symbol.toString())){
```
Comment (romaro488) [inline src/task1/Program.java]: please format your code

File: src/task1/Program.java (hunk 1 above)
Comment (LiuIhor) [reply src/task1/Program.java]: Done. I split the logic of this method into several functions.

File: src/task1/README.txt (hunk 2 above)
Comment (LiuIhor) [reply src/task1/README.txt]: Done

File: src/task1/Program.java (hunk 3 above)
Comment (LiuIhor) [reply src/task1/Program.java]: Done

**Review by romaro488 — APPROVED** [review]

**Review by romaro488 — APPROVED** [review]

**Pull request merged**